CHANGELOG.md

```diff
@@ -0,0 +1,15 @@
+# Changelog
+
+## Unreleased
+### Added
+- New CLI argument `percent-above` for `summarize` crate ([GH-32])
```

Comment:
This should probably note which tool this is a feature of since there's a library and a number of binaries in this repo. Something simple like:

```
### Added
- summarize: New CLI argument ....
```